src/poetry/utils/authenticator.py

```diff
@@ -33,6 +35,26 @@
 logger = logging.getLogger(__name__)
 
 
+class BetterLockFile(lockfile.LockBase):  # type: ignore[misc]
```

Comment:
```suggestion
class FileLockLockFile(lockfile.LockBase):  # type: ignore[misc]
```

I'd like to make it more obvious what the intent is -- otherwise LGTM.

Reply:
I struggle to read that as clearer, but I've no objection to the change...!

Reply:
Clearer might be the wrong word -- painfully explicit and easily grepable is more what I was going for.